network_test.go

```diff
@@ -453,12 +453,22 @@ func startCNIMachine(t *testing.T, ctx context.Context, m *Machine) string {
 }
 
 func testPing(t *testing.T, ip string, count int, timeout time.Duration) {
+	// First, send one ping to make sure the machine is up
 	pinger, err := ping.NewPinger(ip)
 	require.NoError(t, err, "failed to create pinger")
+	pinger.SetPrivileged(true)
+
+	pinger.Count = 1
```

Comment:
Would we want to set a sort of timeout on this pinger also? It looks like the default is pretty high.

Reply:
Sounds good. Will do.

Reply:
Fixed.